vmdb/app/models/ems_refresh/parsers/openstack.rb

```diff
@@ -2,6 +2,7 @@
 
 module EmsRefresh::Parsers
   class Openstack < Cloud
+    include EmsRefresh::Parsers::OpenstackCommon::Images
```

Comment:
I talked to @ladas about this ... he pointed out that the goal is to actually use a module mixin file called `OpenstackCommon` in the `parsers` directory.  So, this is fine, actually...ignore my comment.